src/main/ngapp/accio/src/app/editor/mask.directive.ts

```diff
@@ -193,12 +213,19 @@ export class MaskDirective {
         )
       );
     }
+    else if (this.tool == MaskTool.PAN) {
+      const offsetCoord = this.convertToUnscaledCoord(e.offsetX, e.offsetY);
+      this.newDestinationEvent.emit(
+        new Coordinate((offsetCoord[0] - this.coord[0] + this.translationCoords.x), 
+                       (offsetCoord[1] - this.coord[1] + this.translationCoords.y))
```

Comment:
call helper from above comment.  Anytime you do the same math operation twice it is almost always worth it to create a helper function.